domains/certificates/migrations/API.sql

```diff
@@ -469,3 +469,22 @@ BEGIN
 END $EF$;
 COMMIT;
 
+START TRANSACTION;
+
```

Comment:
Interessant. Dette er teknisk set en migration der er lavet for at rette op på fejlen da vi prøvede at deploye version4 af denne refactor.

Jeg har det ambivalent med at man laver migrations som er designet til at køre en enkelt gang. Denne migration kommer nu til at være en del af vores historik, for evigt. Kan vi ikke blot køre et script en enkelt gang mod prod?